tests/integration/components/o-s-s/information-section-test.ts

```diff
@@ -0,0 +1,109 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupIntl } from 'ember-intl/test-support';
+
+module('Integration | Component | o-s-s/information-section', function (hooks) {
+  setupRenderingTest(hooks);
+  setupIntl(hooks);
+  hooks.beforeEach(function () {
+    this.title = 'Information Section Title';
+    this.subtitle = 'Information Section subtitle';
+    this.namedBlockContent = 'Information Section Custom Content';
+    this.icon = 'fa-pen';
+    this.svgIcon = '/assets/images/upfluence-white-logo.svg';
+    this.actionButton = {
+      label: 'Test Button',
+      icon: 'fa-pen'
+    };
+  });
+
+  test('it renders', async function (assert) {
+    await render(hbs`<OSS::InformationSection @title={{this.title}}/>`);
+
+    assert.dom("[data-control-name='information-section']").exists();
+  });
+
+  test('it renders the @title arg', async function (assert) {
+    await render(hbs`<OSS::InformationSection @title={{this.title}}/>`);
+
+    assert.dom("[data-control-name='information-section--title']").hasText(this.title);
+  });
+
+  test('it renders the @subtitle arg', async function (assert) {
+    await render(hbs`<OSS::InformationSection @title={{this.title}} @subtitle={{this.subtitle}}/>`);
+
+    assert.dom("[data-control-name='information-section--subtitle']").hasText(this.subtitle);
+  });
+
+  module('plain arg', function () {
+    test('it renders the header with a grey background when @plain is false', async function (assert) {
```

Comment:
nit: maybe add a test for the default behavour, when @plain is not specified

Reply:
Added New test